src/main/java/games/strategy/engine/random/PBEMDiceRoller.java

```diff
@@ -276,24 +276,19 @@ private void rollInSeperateThread() throws IllegalStateException {
         appendText("Text from dice server:\n" + text + "\n");
         notifyError();
       } catch (final IOException ex) {
-        try {
-          appendText("An error has occured!\n");
-          appendText("Possible reasons the error could have happened:\n");
-          appendText("  1: An invalid e-mail address\n");
-          appendText("  2: Firewall could be blocking TripleA from connecting to the Dice Server\n");
-          appendText("  3: The e-mail address does not exist\n");
-          appendText("  4: An unknown error, please see the error console and consult the forums for help\n");
-          appendText("     Visit https://forums.triplea-game.org  for extra help\n");
-          if (text != null) {
-            appendText("Text from dice server:\n" + text + "\n");
-          }
-          final StringWriter writer = new StringWriter();
-          ex.printStackTrace(new PrintWriter(writer));
-          writer.close();
```

Comment:
`close()` is a no-op for `StringWriter`.  Removing that call also removes the need to catch the `IOException` it is declared to throw (`ex1` below).